src/transformers/models/maskformer/modeling_maskformer.py

```diff
@@ -2477,7 +2476,7 @@ def forward(
         outputs: MaskFormerModelOutput = self.model(
             pixel_values,
             pixel_mask,
-            output_hidden_states=output_hidden_states,
+            output_hidden_states=output_hidden_states or self.config.use_auxiliary_loss,
```

Comment:
Hmm so if a user set use_auxiliary_loss to True, he will always get hidden states as well? This is not documented anywhere right?